code/modules/client/verbs/ooc.dm

```diff
@@ -468,3 +468,68 @@ ADMIN_VERB(reset_ooc_color, R_FUN, "Reset Player OOC Color", "Returns player OOC
 	set desc = "View the current map vote tally counts."
 	set category = "Server"
 	to_chat(mob, SSmap_vote.tally_printout)
+
+/client/var/forumlinklimit = 0
+/client/verb/linkforumaccount()
+	set category = "OOC"
+	set name = "Link Forum Account"
+	set desc = "Validates your byond account to your forum account. Required to post on the forums."
+
+	var/uri = CONFIG_GET(string/forum_link_uri)
+	if(!uri)
+		to_chat(src, span_warning("This feature is disabled."))
+		return
+
+	if (forumlinklimit > world.time + 100)
+		to_chat(src, {"<span class="userdanger">Please wait 10 game seconds between forums link attempts.</span>"})
+		return
+
+	forumlinklimit = world.time
+
+	if (!SSdbcore.Connect())
+		to_chat(src, {"<span class="danger">No connection to the database.</span>"})
+		return
+
+	if  (is_guest_key(ckey))
+		to_chat(src, {"<span class="danger">Guests can not link accounts.</span>"})
+
+	var/token = generate_account_link_token()
+
+	var/datum/db_query/query_set_token = SSdbcore.NewQuery("INSERT INTO phpbb.tg_byond_oauth_tokens (`token`, `key`) VALUES (:token, :key)", list("token" = token, "key" = key))
+	if(!query_set_token.Execute())
+		to_chat(src, {"<span class="danger">Unknown error #3.</span>"})
+		qdel(query_set_token)
+		return
+
+	qdel(query_set_token)
+
+	to_chat(src, {"Now opening a window to login to your forum account, Your account will automatically be linked the moment you log in. If this window doesn't load, Please go to <a href="[uri]?token=[token]">[uri]?token=[token]</a> This link will expire in 30 minutes."})
```

Comment:
Don't need the brackets

Reply:
I had to escape the quotes for the `a` tags `href` attribute, I assume that's correct? 
`<a href=\"[uri]?token=[token]\">`